tmail-backend/jmap/extensions/src/test/scala/com/linagora/tmail/james/jmap/json/LongLivedTokenSerializerTest.scala

```diff
@@ -0,0 +1,54 @@
+package com.linagora.tmail.james.jmap.json
+
+import com.linagora.tmail.james.jmap.json.Fixture.ACCOUNT_ID
+import com.linagora.tmail.james.jmap.json.LongLivedTokenSerializerTest.{LONG_LIVED_TOKEN_ID, LONG_LIVED_TOKEN_SECRET}
+import com.linagora.tmail.james.jmap.longlivedtoken.{DeviceId, LongLivedTokenId, LongLivedTokenSecret}
+import com.linagora.tmail.james.jmap.model.{LongLivedTokenSetRequest, LongLivedTokenSetResponse, TokenCreateRequest, TokenCreateResponse}
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import play.api.libs.json.{JsResult, JsValue, Json};
+
+object LongLivedTokenSerializerTest {
+  lazy val LONG_LIVED_TOKEN_ID: LongLivedTokenId = LongLivedTokenId.parse("57b8ffe6-4587-4912-886a-6affc83efd90").toOption.get
+  lazy val LONG_LIVED_TOKEN_SECRET: LongLivedTokenSecret = LongLivedTokenSecret.parse("825eee6c-9488-442e-9079-19985c18c532").toOption.get
+}
+
+class LongLivedTokenSerializerTest {
+
+  @Test
+  def deserializeLongLivedTokenSetRequestShouldSuccess(): Unit = {
+    val jsInput: JsValue = Json.parse(
+      """{
+        |    "accountId": "aHR0cHM6Ly93d3cuYmFzZTY0ZW5jb2RlLm9yZy8",
+        |    "create": {
+        |        "deviceId": "My android device"
```

Comment:
Please embed client generated id as per `/set create spec`. What happens when we want to create 2 tokens at once?